web/elm/src/Assets.elm

```diff
@@ -18,6 +18,7 @@ type Asset
     | ChevronRight
     | ToggleSwitch Bool
     | VisibilityToggleIcon Bool
+    | FavoritedToggleIcon Bool
```

Comment:
We already have `StarIconFilled` and `StarIconUnfilled`, should probably just stick to one or the other

Reply:
yeah, also I noticed something today, for this branch, we seemed to have the first two commits with the same commit message. (Somehow that fixup in the rebase doesn't squash the second message with the first one. My bad. I'm trying to fix it now.)

Reply:
Could we maybe have another cleanup pr later (i.e. after issue/5439) for this naming change? Consider the next PR we're working on are using `FavoritedToggleIcon` and `Favorited...` everywhere else. It may cause more rebase for this and next PRs.

Reply:
I think it's probably fine to push the name change down the road a bit, but only if we don't forget about it! Maybe make a note somewhere? (Like in the 5439 issue?)

Reply:
resolved in this PR =)